.envrc

```diff
@@ -0,0 +1,6 @@
+use_node() {
+  local node_version=$(jq -r '.engines.node' package.json)
+  fnm use "$node_version"
```

Comment:
What about nvm? Also, do these tools offer this functionality?

Reply:
It's worth mentioning- for my purposes I use nvm and set the latest as a default alias. Perhaps devs can do something similar then we don't have to concern ourselves with maintaining this for all devs.

Reply:
You're right, this is narrowing the solution. Plus, version resolution is not trivial.
With `.nvmrc` we don't get unpredictable results looking forwards.

Reply:
I use `fnm env --use-on-cd` to get shell automatic update to version. Recommend that you explore fnm.

Reply:
You can use `fnm use --resolve-engines` and it also has a `fnm env --use-on-cd`